tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks/WriteBuildOutputProps.cs

```diff
@@ -33,6 +33,13 @@ public class WriteBuildOutputProps : Task
         /// </summary>
         public ITaskItem[] ExtraPackageInfo { get; set; }
 
+        /// <summary>
+        /// Additional assets to be added to the build output props. They are assumed to have the
+        /// structure <pathToAsset>/<assetName>/<assetVersion>
+        /// i.e. /bin/obj/x64/Release/blobs/Toolset/3.0.100
```

Comment:
Just noticed this comment doesn't really say what the parameter is. The param is just the `<pathToAsset>/<assetName>/` part, and it must have some `<assetVersion>` dir inside it.

Reply:
Clarified this comment (hopefully).

Reply:
I still think it could be clearer (I'd probably open with what the dir/param itself is) but it does describe it once you read the whole thing at least.